libexec/t.rb

```diff
@@ -243,8 +243,8 @@ def is_test_file?(path, suite)
 
   module GitChangedFiles
     def self.cmd(config, test_paths)
-      [ "git diff --no-ext-diff --name-only #{config.changed_ref}", # changed files
-        "git ls-files --others --exclude-standard"                  # added files
+      [ "git diff --no-ext-diff --relative --name-only #{config.changed_ref}", # changed files
+        "git ls-files --others --exclude-standard"                             # added files
```

Comment:
Why do you not need it on the second `git ls-files` command also?

Reply:
It is by default "relative" to the working directly. It has a kind of "opposite" option `--full-name` (https://git-scm.com/docs/git-ls-files#Documentation/git-ls-files.txt---full-name) which returns them based on the git directory.